### PR TITLE
Valid YAML examples

### LINKED ISSUE #152: Guided tour on promotions has bad YAML
If you just copy for example the `production-deploy.yml` from https://docs.semaphoreci.com/article/67-deploying-with-promotions, your pipeline with fail (missing version, maybe also name).

### PR DESCRIPTION
**Pull request by mactsouk**
Valid YAML examples in the guided tour page on promotions.

Closes #152.

### CODE REVIEW

File: tour6-deploying-with-promotions_5bb91da12c7d3a04dd5b5d8f.md [hunk 1]
```diff
@@ -1,36 +1,43 @@
...
-be automatic or manual and optionally depend on conditions.
+On Semaphore 2.0, deployment and delivery is managed with _promotions_, which
+may be performed automatically or manually and may optionally depend on
+user-defined conditions.
 
 ## Manual deployment
 
 Let's start by adding a manual confirmation to promote to production.
 
-<pre><code class="language-yaml"># .semaphore/semaphore.yml
```
Comment (markoa) [inline tour6-deploying-with-promotions_5bb91da12c7d3a04dd5b5d8f.md]: @mactsouk the reason why this was in one line is that if the first line of code snippet starts from line below, it creates too much white space on the live web page.

File: tour6-deploying-with-promotions_5bb91da12c7d3a04dd5b5d8f.md [hunk 2]
```diff
@@ -1,36 +1,43 @@
...
     pipeline_file: production-deploy.yml
 </code></pre>
 
 Now create a new pipeline file in `.semaphore/production-deploy.yml`:
 
-<pre><code class="language-yaml"># .semaphore/production-deploy.yml
+<pre><code class="language-yaml">
+# .semaphore/production-deploy.yml
+version: v1.0
+name: File .semaphore/production-deploy.yml
```
Comment (markoa) [inline tour6-deploying-with-promotions_5bb91da12c7d3a04dd5b5d8f.md]: name should be something descriptive like "Deploy to production".

File: tour6-deploying-with-promotions_5bb91da12c7d3a04dd5b5d8f.md [hunk 3]
```diff
@@ -80,15 +105,31 @@ blocks:
   - name: Deploy
     task:
       jobs:
-        - name: 'Everything'
+        - name: Staging
           commands:
             - echo 'Deploying to staging!'
 </code></pre>
 
 Auto-promotions can also be associated to specific branches. Here's how to
```
Comment (markoa) [inline tour6-deploying-with-promotions_5bb91da12c7d3a04dd5b5d8f.md]: I remember it was hard to find this section, let's add a H3 above: "Continuous deployment on specific branch".

File: tour6-deploying-with-promotions_5bb91da12c7d3a04dd5b5d8f.md [hunk 4]
```diff
@@ -44,10 +51,10 @@ blocks:
             - echo 'Deploying to production!'
 </code></pre>
 
-In the Semaphore web interface, users will see a "Production Deploy" button
-once all blocks in the pipeline defined in `semaphore.yml` pass.
-When a user promotes a revision, Semaphore records the time and name of the
-person who initiated it, and proceeds by executing the pipeline defined in
+In the Semaphore 2.0 web interface, users will see a `Production Deploy` button
```
Comment (markoa) [inline tour6-deploying-with-promotions_5bb91da12c7d3a04dd5b5d8f.md]: Let's not put that in code block please. It appears in UI, not code or terminal output.

File: tour6-deploying-with-promotions_5bb91da12c7d3a04dd5b5d8f.md [hunk 5]
```diff
@@ -1,36 +1,43 @@
-Each project starts with the default pipeline specified in
+Each Semaphore 2.0 project starts with the default pipeline specified in
```
[the remark this answers is not in the document]
Comment (mactsouk) [reply tour6-deploying-with-promotions_5bb91da12c7d3a04dd5b5d8f.md]: OK.

File: tour6-deploying-with-promotions_5bb91da12c7d3a04dd5b5d8f.md (hunk 1 above)
Comment (mactsouk) [reply tour6-deploying-with-promotions_5bb91da12c7d3a04dd5b5d8f.md]: Yes, but this way leaves a space character in front of the first line.

File: tour6-deploying-with-promotions_5bb91da12c7d3a04dd5b5d8f.md (hunk 2 above)
Comment (mactsouk) [reply tour6-deploying-with-promotions_5bb91da12c7d3a04dd5b5d8f.md]: Done.

File: tour6-deploying-with-promotions_5bb91da12c7d3a04dd5b5d8f.md (hunk 3 above)
Comment (mactsouk) [reply tour6-deploying-with-promotions_5bb91da12c7d3a04dd5b5d8f.md]: Done.

File: tour6-deploying-with-promotions_5bb91da12c7d3a04dd5b5d8f.md (hunk 4 above)
Comment (mactsouk) [reply tour6-deploying-with-promotions_5bb91da12c7d3a04dd5b5d8f.md]: Done.

File: tour6-deploying-with-promotions_5bb91da12c7d3a04dd5b5d8f.md [hunk 6]
```diff
@@ -44,10 +51,10 @@ blocks:
             - echo 'Deploying to production!'
 </code></pre>
 
-In the Semaphore web interface, users will see a "Production Deploy" button
-once all blocks in the pipeline defined in `semaphore.yml` pass.
-When a user promotes a revision, Semaphore records the time and name of the
-person who initiated it, and proceeds by executing the pipeline defined in
+In the Semaphore 2.0 web interface, users will see a `Production Deploy` button
+once all blocks in the pipeline defined in `semaphore.yml` pass. When a user
```
[the remark this answers is not in the document]
Comment (mactsouk) [reply tour6-deploying-with-promotions_5bb91da12c7d3a04dd5b5d8f.md]: Done.

**Pull request merged**